### PR TITLE
Adding state-related tokens for notifications

### PR DESCRIPTION
**Pull request by StefanIvanov**
After suggestions from Iveto and @kckfm1

### CODE REVIEW

File: data/tokens.json [hunk 1]
```diff
@@ -2797,6 +2791,10 @@
       }
     },
     "Banner": {
+      "gtmds_banner-dismiss-button_background-color-hover": {
+        "value": "rgba({Base Notifications.gtmds_notification_color},0.1)",
```
Comment (Ddungyov) [inline data/tokens.json:2795]: wow, does this thing work :D ?

File: data/tokens.json [hunk 2]
```diff
@@ -2887,10 +2895,18 @@
         "value": "{gtmds_color_white}",
         "type": "color"
       },
-      "gtmds_toast-close-icon_color": {
+      "gtmds_toast-dismiss-button_color": {
         "value": "rgba({Toast.gtmds_toast_color}, 0.5)",
```
Comment (Ddungyov) [inline data/tokens.json:2899]: I saw this color is used with opacity, while we were discussing initially to avoid opacity. What is the reason for that?

Comment (StefanIvanov) [reply data/tokens.json:2899]: reason is that we have very colorful situation and want to introduce one thing that would work everywhere but we need opacity as we don't want a color that is as strong as the rest of the content

File: data/tokens.json (hunk 1 above)
Comment (StefanIvanov) [reply data/tokens.json:2795]: the plugin supports the css functions so in theory, yes it should work out of the box without extra effort on our end

**Pull request merged**